travis/Dockerfile-qt5-travis

```diff
@@ -0,0 +1,15 @@
+FROM openms/contrib:qt5-v2.0
+
+WORKDIR /
+ADD . /OpenMS
+RUN cd /OpenMS 
```

Comment:
nice :+1: so do we have an openms branch where we try this? I guess its something like this:

https://docs.travis-ci.com/user/docker/

with


```
sudo: required

services:
  - docker
```